JS/Lesson16/sinan/script.js

```diff
@@ -1,142 +1,177 @@
 // 1. Write a loop that prints out the numbers from 1 to 10.
-for (let i = 1; i<=10; i++) {
+for (let i =0; i<10; i++) {
 	console.log(i);
-}
+	}
 console.log('- - - - - - - - - - - - - - - - - ');
 // 2. Write a loop that prints out the odd numbers from 1 to 20.
 // 1,3,5,7...
-for (let i = 1; i<20; i = i+2){
+for(let i=1; i<20; i+=2){
 	console.log(i);
 }
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 3. Write a loop that prints out the even numbers from 1 to 20.
 // 2, 4, 6...
-for (let i =2; i<=20; i+=2) {
+for(let i =0; i<20; i+=2) {
 	console.log(i);
 }
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 4. Write a loop that finds the maximum number in an array of numbers.
-const findMaxNumberInArray = [300, 10, 23, 42, 505, 12, 7, 4, 900];
+const findMaxNumberInArray = [30, 10, 23, 42, 5505, 12, 77777, 4, 900,12300];
 let num = findMaxNumberInArray[0];
-for(let i = 0; i<findMaxNumberInArray.length; i++) {
-if(num<findMaxNumberInArray[i+1]){
+for (let i = 0; i < findMaxNumberInArray.length; i++){
+	if(num < findMaxNumberInArray[i+1]){
 	num = findMaxNumberInArray[i+1];
-}
-}
-console.log(num);
+}}
+console.log('Exercise 4:',num);
 
 console.log('- - - - - - - - - - - - - - - - - ');
 // 5. Write a loop that finds the minimum number in an array of numbers.
-const findMinNumberInArray = [1, 10, 23, 42, 0, 12, 7, 0];
-let num1 = findMinNumberInArray[0];
-
-for (let i= 1; i<findMinNumberInArray.length; i++) {
-	if(num1>findMinNumberInArray[i]) {
-			num1 = findMinNumberInArray[i];
+const findMinNumberInArray = [1, 10, 23, 42, 5, 12, 7, -2,-15];
+let minNumber = findMinNumberInArray[0];
+for (let i = 0; i < findMinNumberInArray.length; i++) {
+	if(minNumber > findMinNumberInArray[i+1]){
+		minNumber = findMinNumberInArray[i+1]
 	}
 }
-console.log(num1);
+console.log('Exercise 5:',minNumber);
+
 console.log('- - - - - - - - - - - - - - - - - ');
 // 6. Write a loop that reverses a given string.
 const string = 'Lorem ipsum dolor sit amet consectetur adipisicing elit.';
-let newString = "";
-for (let i = string.length - 1; i >= 0; i--) {
-newString = newString + string[i]; 
-// . + t 
-// .t + i
-// .ti + l
+let newString = ""; 
+for ( let i = string.length-1; i>=0; i--){
+	newString = newString + string[i];
 }
-console.log(newString);
+console.log('Exercise 6:',newString);
 
-const arrayEx6 = [1,2,3,5,242,32,23,232,112,233];
-let newArray = [];
 
-for(let i = arrayEx6.length-1; i >= 0; i--){
+const arrayEx6 = [1,2,3,4,5,6,7,89,8,6,3,23,324,123];
+let newArray = [];
+for (let i=arrayEx6.length-1; i >=0; i--)
+{
 	newArray.push(arrayEx6[i]);
 }
-console.log(newArray);
-
+console.log('Exercise 6.1:',newArray);
 console.log('- - - - - - - - - - - - - - - - - ');
 // 7. Write a while loop that counts from 10 to 1 and prints out each number.
-let i = 10;
-while(i>=1) {
-	console.log(i);
-	i--;
+let count = 0;
```

Comment:
It should be the other way around: from 10 to 1, the code that was here before was correct :)